### ISSUE TITLE
document array rule key not present in json

### ISSUE THREAD

**Issue by Maatteogekko** [appwrite/appwrite#1275]
I have a collection with a rule name "todos" which is an array of documents. If I don't have any document the json response does not contain the key "todos", when instead it should contain something like `"todos": []`.
![image](https://user-images.githubusercontent.com/54111322/121657255-89c68900-caa0-11eb-938d-4781d32dfbb5.png)

```json
{
    "$id": "60c21f5e3879c",
    "$collection": "60b8933b7766e",
    "$permissions": {
        "read": [
            "*"
        ],
        "write": [
            "*"
        ]
    },
    "body": "some text",
    "color": 1,
    "timeStamp": 1
}
```

---
Appwrite Version: 0.8.0.146

**Comment by eldadfux** [comment]
This is probably an unexpected behavior. We will work on fix it as part of the upcoming database updates RFC: https://github.com/appwrite/rfc/pull/21

**Comment by eldadfux** [comment]
This issue was fixed and its now available as part of Appwrite 0.12. You can learn more about the new version here: https://dev.to/appwrite/its-here-announcing-the-release-of-appwrite-012-5c8b
